cypress/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
are we using typescript here? is this file actually use?

Reply:
this is for suggestion during coding
https://docs.cypress.io/guides/tooling/intelligent-code-completion.html#Set-up-in-your-Dev-Environment